mkat_fpga_tests/test_cbf.py

```diff
@@ -1470,3 +1470,51 @@ def test_all_delays(self):
             Aqf.less(abs_diff, 1,'Check that the maximum degree between '
                 'expected and actual phase difference between intergrations '
                 'is below 1 degree: {0:.3f} degree\n'.format(abs_diff))
+    @aqf_vr('TP.C.1.17')
+    def test_config_report(self):
+        """CBF Report configuration"""
+
+        repos = ['corr2', 'mkat_fpga', 'casperfpga', 'mkat_fpga_tests',
+                 'katcp-python']
+        src_dir = '/usr/local/src'
+        CMD = subprocess.check_output
+        Aqf.step('CMC CBF Package Software version information')
+        for repo in repos:
+            git_hash = CMD(['git', '--git-dir={}/{}/.git'.format(src_dir, repo),
```

Comment:
By the way, did you ever look at Steve's system information python scripts? Might also have other useful information that you can extract, or perhaps he's done some other potentially useful work for you.